src/libraries/System.Runtime.Loader/tests/System.Runtime.Loader.Test.AssemblyVersion3/System.Runtime.Loader.Test.AssemblyVersion3.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
The `System.Runtime.Loader.Test.AssemblyVersion3` project does not actually seem to be needed. The tests only need to request version 3.0.0. They never actually need to load a 3.0.0 version.